pom.xml

```diff
@@ -1658,6 +1668,17 @@
                 <artifactId>curator-recipes</artifactId>
                 <version>4.0.1</version>
             </dependency>
+            <!-- standardize httpcomponents version, used by various libraries including hadoop and thrift -->
+            <dependency>
+                <groupId>org.apache.httpcomponents</groupId>
+                <artifactId>httpclient</artifactId>
+                <version>4.5.3</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.httpcomponents</groupId>
+                <artifactId>httpcore</artifactId>
+                <version>4.4.6</version>
```

Comment:
The versions of httpclient and httpcore in the Hadoop/Apache ecosystem don't match?

Reply:
it's not hadoop, for some reason the two artifacts are on different versioning schemes: https://search.maven.org/search?q=g:org.apache.httpcomponents

Reply:
Good to know.